api/controllers/FlairController.js

```diff
@@ -154,7 +154,7 @@ module.exports = {
 
       for (var i = 0; i < flairs.fcs.length; i++) {
         let fc = flairs.fcs[i];
-        if (!Flairs.validFC(fc) && _.contains(req.user.loggedFriendCodes, fc)) {
+        if (!Flairs.validFC(fc) && flairs.fcs[i] !== req.user.loggedFriendCodes[i]) {
```

Comment:
Personally I think that's best -- we should only filter these reports when they're providing no new information (e.g. a user sets their flair text again with the same invalid friend code). It makes sense to send a modmail when a user changes from valid FC to invalid FC, since that's a suspicious activity that mods might want to keep an eye on.